packages/core-mobile/app/store/browser/const.ts

```diff
@@ -1,74 +1,74 @@
-import { ImageRequireSource } from 'react-native'
-
 export const MAXIMUM_TABS = 100
 export const MAXIMUM_TAB_HISTORIES = 20
 export const MAXIMUM_HISTORIES = MAXIMUM_TABS * MAXIMUM_TAB_HISTORIES
 
 export type SugggestedItem = {
-  name: string
+  name: SuggestedLogo
   siteUrl: string
-  logo: ImageRequireSource
+}
+
+export enum SuggestedLogo {
```

Comment:
This enum seems to contain names rather than logos, should it then be called SuggestedSiteName or similar?

Reply:
that's true. 👍

Reply:
update [here](62150fa3fe719cd2b2d25aaad7993b0d6ab3051a)